web/packages/dmt-core/src/components/Pickers/JobHandlerPicker.tsx

```diff
@@ -8,6 +8,7 @@ export const JobHandlerPicker = (props: {
 }) => {
   const { onChange, formData } = props
   const blueprintName = formData.split('/').pop()
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore